roles/common/tasks/main.yml

```diff
@@ -85,6 +86,18 @@
   file: name=/var/lib/rsyslog state=directory
   tags: rsyslog
 
+- name: Put rsyslog client cert
+  copy: content="{{ rsyslog_certificate }}" dest=/etc/pki/rsyslog/rsyslogclient.crt mode=0744
+  tags: rsyslog
+
+- name: Put rsyslog client key
+  copy: content="{{ rsyslog_key | vault(vault_keydir) }}" dest=/etc/pki/rsyslog/rsyslogclient.key owner=root mode=400
+  tags: rsyslog
+  no_log: true
```

Comment:
AFIK `no_log` is not necessary. The content (i.e. the decryped key) is not logged.